vips/vip-304/bscmainnet.ts

```diff
@@ -0,0 +1,40 @@
+import { parseUnits } from "ethers/lib/utils";
+
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+export const VUSDC = "0xecA88125a5ADbe82614ffC12D0DB554E2e2867C8";
+export const TOKEN_REDEEMER = "0x13fFde8050fa0Ef5A6f3c28B500c9267ec8A2C46";
```

Comment:
We can use the last TokenRedeemer contract, deployed at `0xC53ffda840B51068C64b2E052a5715043f634bcd`. See it [here](https://github.com/VenusProtocol/venus-protocol/pull/482). This TokenRedeemer contract includes a function `redeemUnderlyingAndTransfer`, that would allow us to transfer an exact amount of underlying tokens

Reply:
Done https://github.com/VenusProtocol/vips/pull/286/commits/af5b2d8529952b3733696c1ed7f7925784994c75